Localization/Translations/DebugMessages.lua

```diff
@@ -55,7 +55,7 @@ local debugMessagesLocales = {
         ["ruRU"] = "состояние трекера Questie. Добавьте 'show', 'hide', 'reset', чтобы показать, скрыть или сбросить трекер.",
         ["deDE"] = "Zeigen/Verstecken des Questie Trackers. Bei 'show', 'hide' oder 'reset' wird der Tracker explizit gezeigt, versteckt oder zurückgesetzt",
         ["koKR"] = nil,
-        ["esMX"] = nil,
+        ["esMX"] = "Muestra el Rastreador. Agrega 'mostrar', ''ocultar y 'reestablecer' para mostrar/ocultar y reestablecer el Rastreador",
```

Comment:
There are two `'` which doesn't seem right.

Reply:
You're absolutely right, fixing it. I kinda messed up the merge, that's why I created a new PR, but I might take the offer to soak some more knowledge on how to be more proficient at this.